src/dsp_tools/commands/excel2json/properties.py

```diff
@@ -166,7 +166,7 @@ def _add_optional_columns(df: pd.DataFrame) -> pd.DataFrame:
     in_df_cols = set(df.columns)
     if not optional_col_set.issubset(in_df_cols):
         additional_col = list(optional_col_set.difference(in_df_cols))
-        additional_df = pd.DataFrame(columns=additional_col, index=df.index, data=None)
+        additional_df = pd.DataFrame(columns=additional_col, index=df.index)
```

Comment:
The problem is now fixed.
https://chat.openai.com/share/e/f7229856-04fb-407b-bee5-c6d6b986be60
I didn't realise that if you give an index, you are also allowed to leave data empty...